### PR TITLE
Allow labeling of response/benefit labels

### PR DESCRIPTION
**Pull request by arahuja**
Converts the x-axis to look like
image

### CODE REVIEW

File: cohorts/load.py [hunk 1]
```diff
@@ -1219,7 +1253,6 @@ def summarize_provenance_per_cache(self):
         frame for this cohort.
         """
         provenance_summary = {}
-        df = self.as_dataframe()
```
Comment (arahuja) [inline cohorts/load.py:1222]: I was removing this as they showed up as unused, but now seeing that we are relying on a side-effect of these functions to fill in the value?

**Comment by arahuja** [comment]
These PR originally dropped some used call to `as_dataframe()` but now seeing that we are relying on a side-effect of these functions to fill in the value?

File: cohorts/load.py [hunk 2]
```diff
@@ -1114,7 +1135,18 @@ def plot_boolean(self, on, boolean_col, col=None, ax=None,
...
         df = filter_not_null(df, plot_col)
-        df[boolean_col] = df[boolean_col].astype(bool)
+
+        if boolean_label:
+            df[boolean_label] = df[boolean_col]
+            boolean_col = boolean_label
+
+        condition_value = None
+        if boolean_value_map:
+            assert (True in boolean_value_map) and (False in boolean_value_map), \
```
Comment (tavinathanson) [inline cohorts/load.py]: Since you also want these to be the only keys, maybe `assert set(boolean_value_map.keys()) == set([True, False])`?

**Comment by tavinathanson** [comment]
That seems confusing re `as_dataframe`. I'll file an issue. Otherwise LGTM!

**Comment by arahuja** [comment]
Thanks, updated, the assert, will merge on Travis green-light

**Pull request merged**